src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,75 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+
+/**
+ * Represents Duke's storage system and handles file operations.
+ */
+public class Storage {
+    //private static List<String> lines;
```

Comment:
I'm guessing you intend to remove this in the end? Since it's not really a comment that describes the code but probably an intermediate step you took?